src/test/java/org/junit/samples/ListTest.java

```diff
@@ -35,9 +35,9 @@ public static void setUpOnce() {
     public void setUp() {
         fEmpty = new ArrayList<Integer>();
         fFull = new ArrayList<Integer>();
-        fFull.add(1);
-        fFull.add(2);
-        fFull.add(3);
+        for (int i = 1; i < 4; i++) {
```

Comment:
I personally prefer to avoid loops in tests. Tests, especially unit tests, should be simple and obviously correct.

In this particular case, the `for` loop doesn't follow the standard idiom for looping `n` times. It does so for a good reason, but that makes the code slightly more confusing.